ui/packages/catalog-ui-search/src/main/webapp/component/map-context-menu/map-context-menu.hbs

```diff
@@ -1,18 +1,17 @@
 {{!--
 /**
- * Copyright (c) Codice Foundation
- *
- * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
- * version 3 of the License, or any later version.
- *
- * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
- * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
- * <http://www.gnu.org/licenses/lgpl.html>.
- *
- **/
- --}}
-<div class="metacard-interaction interaction-copy-coordinates" data-help="Copies the coordinates to your clipboard."
-     data-clipboard-text="{{this.clickLat}} {{this.clickLon}}">
+* Copyright (c) Codice Foundation
+*
+* This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation, either
+* version 3 of the License, or any later version.
+*
+* This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+* See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is
+distributed along with this program and can be found at
+* <http://www.gnu.org/licenses/lgpl.html>. * **/ --}} <div class="metacard-interaction interaction-copy-coordinates"
```

Comment:
move the div to the next line

Reply:
Can't this is how Prettier, our formatter, arranges it

Reply:
Try putting `--}}` on its own line too
https://github.com/codice/ddf/blob/3121ca78db74b48d3ff087a8a6fcff8fe2e75bcf/ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/search-form-interactions/dropdown.search-form-interactions.hbs#L12-L14